### PR TITLE
fix: copy change french translation inputs

### LINKED ISSUE #1485: Copy change: French translation inputs
**Prerequisites**

- [x] I've [searched](https://github.com/accessibility-exchange/platform/issues) for existing issues

**Describe the bug**
Bug not addressed in #1419:
When FROs add translation, change the button label to “Add French translation of …”

### PR DESCRIPTION
**Pull request by chosww**
### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [ ] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: app/strings.php [hunk 1]
```diff
@@ -186,12 +186,12 @@
             __('Preview'),
             __('Publish project'),
             __('Once you publish your project, others can find it.'),
-            __(':language translation of project goals'),
+            __('Add :language translation of project goals'),
```
Comment (jobara) [inline app/strings.php]: I wonder if all/many of these can be removed. Do you know if the parts after "Add :language translation of" should have been token for interpolation like "Add :language translations of :label" and is already handled in the translatable-input.blade.php and translatable-textarea.blade.php files.

**Review by jobara — CHANGES_REQUESTED** [review]

File: app/strings.php (hunk 1 above)
Comment (chosww) [reply app/strings.php]: Right, I think parts after "Add :language translation of" are handled by translatable components; the components use short label or label property for the after part, and current use cases all provide either label or short label.

**Comment by jobara** [comment]
@chosww looks like there are some merge conflicts with dev now. Could you please update as well?

File: app/strings.php (hunk 1 above)
Comment (jobara) [reply app/strings.php]: @chosww thanks for confirming that. I think we should remove these strings from the string placeholder file then, if they aren't necessary and are already handled.

**Review by jobara — APPROVED** [review]

**Pull request merged**